### PR TITLE
added key fingerprint in .ssh documentation

### PR DESCRIPTION
**Pull request by mckeags7**

### CODE REVIEW

File: docs/connecting/ssh.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ Finally, in the main screen again, save the profile you created so you don't hav
 
 ![PuTTY Save](res/putty-save.png)
 
+The server's ssh-ed25519 key fingerprint is:
+ssh-ed25519 255 SHA256:jC7BF7h5/RJo5Svx1v+lufdf+I/ogu5dQV2sUe+y8ek
```
Comment (hakasapl) [inline docs/connecting/ssh.md:60]:
```suggestion
`ssh-ed25519 255 SHA256:jC7BF7h5/RJo5Svx1v+lufdf+I/ogu5dQV2sUe+y8ek`
```

Wrap these in tickmarks so they show up as a code block

**Review by hakasapl — CHANGES_REQUESTED** [review]
Just the tickmarks change, otherwise looks good

**Comment by hakasapl** [comment]
Github doesn't like me including "```" in the suggestions, but just wrap the whole fingerprint with three tick marks at the start and the end so it becomes a code block.

**Review by simonleary-umass-edu — CHANGES_REQUESTED** [review]
image

without a code block the text is mashed into one line.
Do we really need an example? It's just a notepad window surrounding a copy of the text we already provided. We could just reword it to tell them to copy it into notepad, rather than "append the contents"

File: docs/connecting/ssh.md [hunk 2]
```diff
@@ -37,35 +37,22 @@ If you know what to do with this information already, you can skip the rest of t
 !!! note
     Your username should be your email with all at's `@` and dot's `.` replaced with underscores `_`. View your username [here](https://unity.rc.umass.edu/panel/account.php)
```
Comment (hakasapl) [inline docs/connecting/ssh.md:38]: This is not strictly correct. Often times this is true, but we use EPPN (Edu-person principal name) to create usernames, and sometimes people will have emails that are not the same as their netids. (I know this isn't part of this PR but I just spotted it, so could you update that as well?)

```suggestion
    Your username is in the format `<organization username>_<organization>_edu`. View your username [here](https://unity.rc.umass.edu/panel/account.php)
```

**Review by hakasapl — CHANGES_REQUESTED** [review]

**Review by hakasapl — APPROVED** [review]
Looks good to me, thanks!

**Review by bryank-cs — APPROVED** [review]

**Review by simonleary-umass-edu — APPROVED** [review]
